mlir/include/mlir/Dialect/Linalg/Transforms/Transforms.h

```diff
@@ -1415,6 +1416,7 @@ struct DownscaleSizeOneWindowed2DConvolution final
                                 PatternRewriter &rewriter) const override {
     return returningMatchAndRewrite(convOp, rewriter);
   }
+  using OpRewritePattern<Conv2DOp>::matchAndRewrite;
```

Comment:
Wouldn't it be better to have these protected like in all other places?

Reply:
Honestly, I think we at least don't care that the other variants are hidden, and don't necessarily want them here public or protected. They're public in the base class, I believe, but I don't suppose we want them to be called, so probably protected is better.

Or maybe this is an argument for just adding `-Wno-overloaded-virtual` to the ClangIR build, since apart from the warning the current behavior is what we want.